docs/DeveloperGuide.md

```diff
@@ -6,33 +6,230 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### The Overall UML Diagram:
 
+![UML Diagram](assets/images/mathGenius.png)    
+
+
+### The overall Sequence Diagram:
+![Sequence Diagram](assets/images/sequenceDiagram.png)    
+
+
+
+### problemGenerator Component
+API: [ProblemGenerator.java](../src/main/java/seedu/duke/ProblemGenerator.java)
+
+the main idea in the design of problemGenerator is that it take 4 parameters and generate a test which include some number of problems
+
+
+when the gengerator is triggered, users enter a command like this:
+
+    generate -t [operators] -n [number] -d [maximum digit] -l[length]
+for example , user can enter:
```

Comment:
https://ay2324s2-cs2113-t13-1.github.io/tp/DeveloperGuide.html
wrong display? the "for example, user can enter: " part is inside the code blocks